components/ui/Footer.js

```diff
@@ -0,0 +1,87 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import Link from 'next/link'
+import { StyledLinkText, Body1 } from './Typography'
+import theme from '../../util/theme'
+
+const FooterContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 8rem;
+
+  background: ${props => props.backgroundColor}
+    ${props => props.theme.halftones.md};
+`
+
+const FooterLinksContainer = styled.div`
+  display: flex;
+  font-size: 2rem;
+  line-height: 2rem;
+
+  > :not([hidden]) ~ :not([hidden]) {
+    border-right-width: 0;
+    border-left-width: 0.1rem;
+    border-left-style: solid;
+    border-color: ${theme.colors.white};
+  }
+`
+
+const FooterLink = styled.div`
```

Comment:
Hmm.... (with the notes that these are all now external links) you could just `const A = styled.a` now...